WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderActivityLauncher.java

```diff
@@ -35,20 +36,26 @@
      * with a single post
      */
     public static void showReaderPostDetail(Context context, long blogId, long postId) {
-        showReaderPostDetail(context, false, blogId, postId, false);
+        showReaderPostDetail(context, false, blogId, postId, null, 0, false, null);
     }
 
     public static void showReaderPostDetail(Context context,
```

Comment:
The number of parameters here is getting quite large. I think it would be a good idea to have a comment here stating the purpose of each (esp `directOperation` and `interceptedUri`).